api/coreservice.go

```diff
@@ -306,14 +306,17 @@ func newCoreService(
 }
 
 // Account returns the metadata of an account
-func (core *coreService) Account(addr address.Address) (*iotextypes.AccountMeta, *iotextypes.BlockIdentifier, error) {
+func (core *coreService) Account(addrStr string) (*iotextypes.AccountMeta, *iotextypes.BlockIdentifier, error) {
 	ctx, span := tracer.NewSpan(context.Background(), "coreService.Account")
 	defer span.End()
-	addrStr := addr.String()
 	if addrStr == address.RewardingPoolAddr || addrStr == address.StakingBucketPoolAddr {
 		return core.getProtocolAccount(ctx, addrStr)
 	}
 	span.AddEvent("accountutil.AccountStateWithHeight")
+	addr, err := address.FromString(addrStr)
```

Comment:
addrStr is 0x encode format from web3 request, instead of io encode

Reply:
it is converted to ioAddress before calling `coreService.Account()`